spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinder.java

```diff
@@ -152,19 +147,17 @@ protected final String groupedName(String name, String group) {
 	}
 
 	protected final MessageValues serializePayloadIfNecessary(Message<?> message) {
```

Comment:
Given we don't do any serialization at the `AbstractBinder` , maybe we can get rid of this method completely and have simple method or a step to do the transformation from `Message` to `MessageValues` ?

Reply:
Please see previous comment